ast/Trees.cc

```diff
@@ -811,6 +811,8 @@ string Literal::toStringWithTabs(const core::GlobalState &gs, int tabs) const {
     string res;
     typecase(
         this->value, [&](const core::LiteralType &l) { res = l.showValue(gs); },
+        [&](const core::LiteralIntegerType &i) { res = i.showValue(gs); },
+        [&](const core::FloatLiteralType &i) { res = i.showValue(gs); },
```

Comment:
Literal Integer

vs

Float Literal

=> Maybe we should pick a consistent order?